utils/common/src/test/java/org/apache/brooklyn/util/stream/StreamGobblerTest.java

```diff
@@ -29,12 +29,65 @@
 
 import org.apache.brooklyn.test.Asserts;
 import org.apache.brooklyn.util.os.Os;
-import org.apache.brooklyn.util.stream.StreamGobbler;
+import org.apache.brooklyn.util.text.Strings;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 public class StreamGobblerTest {
+
+    private static final Logger LOG = LoggerFactory.getLogger(StreamGobblerTest.class);
+
     private String NL = Os.LINE_SEPARATOR;
```

Comment:
@algairim Make this final.

Reply:
Addressed in ba780b1acac6c42b4d4f5cc8aa0d063339869847.